int64 flighttime
---
mavros_msgs/Waypoint[] waypoints
